GETTING_STARTED.md

```diff
@@ -909,6 +909,9 @@ end
 Note that this approach works with `build`, `build_stubbed`, and `create`, but
 the associations will return `nil` when using `attributes_for`.
 
+Also, note that if you assign any attributes inside a custom `initialize_with` (e.g. `initialize_with { new(**attributes) }`), those attributes should not refer to `instance`, since it will be `nil`.
+objects are initialized via `initialize_with { new(attributes) }`.
```

Comment:
Oops, it looks like we ended with an extra partial sentence here. Once we get rid of that this should be good to merge. Thanks again!

Reply:
Damn, sorry. Fixed that!